packages/ui/src/components/Organisms/Card.tsx

```diff
@@ -0,0 +1,65 @@
+import { useIntl } from '@amazeelabs/react-intl';
+import { CardItemFragment, Image, Link } from '@custom/schema';
+import React from 'react';
+
+export const CardItem = ({
```

Comment:
File is named `Card.tsx` but we export a `CardItem`, would be good to name one or the other ways?